mbed_host_tests/host_tests_plugins/host_test_plugins.py

```diff
@@ -137,13 +137,15 @@ def check_mount_point_ready(self, destination_disk, init_delay=0.2, loop_delay=0
                 # mbed_lstools.create() should be done inside the loop.
                 # Otherwise it will loop on same data.
                 mbeds = mbed_lstools.create()
-                mbeds_by_tid = mbeds.list_mbeds_by_targetid()   # key: target_id, value mbedls_dict()
-                if target_id in mbeds_by_tid:
-                    if 'mount_point' in mbeds_by_tid[target_id]:
-                        if mbeds_by_tid[target_id]['mount_point']:
-                            # Only assign if mount point is known (not None)
-                            new_destination_disk = mbeds_by_tid[target_id]['mount_point']
-                            break
+                mbed_list = mbeds.list_mbeds(unique_names=True, read_details_txt=True) #list of mbeds present
```

Comment:
I don't think that you need the `read_details_text` and could use the default of False here. You're not using the `platform_unique_id`, so you can also drop `unique_names`.

Reply:
fixed in last commit